src/chrome/Link.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import classNames from 'classnames'
+import { Link } from 'react-router-dom'
```

Comment:
use `as RouterLink` so you can get the `Link` name back:

```ts
import { Link as RouterLink } from 'react-router-dom`
```

rename `CustomLink` -> `Link`